geopandas/tests/test_geom_methods.py

```diff
@@ -395,9 +395,14 @@ def test_exterior(self):
 
     def test_interiors(self):
         square_series = GeoSeries(self.nested_squares)
-        exp_interiors = GeoSeries([LinearRing(self.inner_sq.boundary)])
-        for expected, computed in zip(exp_interiors, square_series.interiors):
-            assert computed[0].equals(expected)
+        square_series = self.g0
+
+        expected = []
+        assert list(square_series.interiors[0]) == expected
+        expected = LinearRing(self.inner_sq.boundary)
```

Comment:
I would also find comments on each of these three tests useful. Since the behavior is so different in each case, a note "this is a polygon with no interior; this is a polygon with an interior; this is a point for which interior is not a valid operation".

Reply:
Thanks for your review.
Test certainly was poor quality.
I would like to fix it.